content/docs/SUSHI/running/_index.md

```diff
@@ -227,6 +227,12 @@ First, you must [install Firely Terminal](https://docs.fire.ly/projects/Firely-T
 You can see a list of the available Firely Terminal versions [here](https://www.nuget.org/packages/Firely.Terminal#versions-body-tab). Version 3.1.0 or higher is required to use the `inflate` command. Versions prior to 3.1.0 offer similar functionality via a `bake` command, but they are no longer recommended.
 {{% /alert %}}
 
+{{% alert title="Firely Terminal Note" color="info" %}}
+Inflating a single package, as described here with `fhir inflate --package  <packagename>`, is a free option in Firely Terminal. Using `fhir install <packagename>` will download, install, and automatically "inflate" the package. Using `fhir inflate` as a bulk option to inflate the current project and all its dependencies requires a Simplifier license.
+
+The [Package Management section](https://docs.fire.ly/projects/Firely-Terminal/Managing-Packages.html) of the Firely Terminal documentation provides additional details.
```

Comment:
I don't recall, do we have a way to launch links in a separate tab/window? If so, can we / should we do that here?

Reply:
I don't recall either, but that's a good idea so I'll check it out.

Reply:
Done - [e4248a1](https://github.com/FSHSchool/site/pull/79/commits/e4248a1047acdf7148fdeae0c2c16f20137dd98c).